### ISSUE TITLE
Fire and Spiderwebs

### ISSUE THREAD

**Issue by sarinilla** [ricegf/elven-fire#17]
Rules are needed for casting Fireball into a webbed room.

Our on-the-spot ruling was that Fireball clears one hex of web per point invested.  (Presumably this means that casting a 3-point Fireball at a spider hiding behind two hexes of webbing results in two cleared hexes and a one-point fireball actually hitting the spider.)

**Comment by JaceR** [comment]
I disagree slightly. (Hey! Stop throwing fireballs!) Spiderweb has a Strength, although perhaps that should be altered for fire. I think a really strong Fireball (d12 for 20 IQ) would blast right through the first web and burn out the second, at least. While a small, weak fireball (d4 for instance) would burn the first web, but certainly not blast through it.

Perhaps tie the amount of web it burns through to the level or power of the ability.
